### DISCUSSION TITLE
"More/Less than or equal to" filters out equal to?

### DISCUSSION THREAD

**Discussion by antm** [Budibase/budibase#11975] [Help]
I'm kicking the tires and following the the filtering, sorting, searching video on Budibase University. I implemented a more than or equal to and less than or equal to form to filter a table but I'm not getting _equal to_ rows to show.

My filter setup: 
<img width="867" alt="image" src="https://github.com/Budibase/budibase/assets/191669/f4fc5799-a91b-4538-8be5-7c15cc16e580">

The row that should show up when filtered by income:
![image](https://github.com/Budibase/budibase/assets/191669/fa4d2ad9-858b-4fe3-8f07-87198cf009bd)

What is returned:
![image](https://github.com/Budibase/budibase/assets/191669/70830414-7197-4f1b-96ec-9f4d4095d9ea)

The income field is a number field (bigint in this case). Did I miss a step?

**Comment by ConorWebb96** [comment]
Hey @antm,

What version of Budibase are you running?
Are you self-host or cloud?
What data source are you using? Is it the internal datasource or are you using an external datasource?

Have you definitely setup your filters correctly? I've setup my filters and everything seems to be work as expected. 
<img width="883" alt="Screenshot 2023-10-09 at 09 06 53" src="https://github.com/Budibase/budibase/assets/126772285/e46ed471-b60f-46b4-82c4-9c26b1efaf48">

Heres a demo
![Screen Recording 2023-10-09 at 09 14 56](https://github.com/Budibase/budibase/assets/126772285/d42a5a3f-7f31-4552-a7ee-38b98a344bb9)

I'm happy to send across an app export if you are using the internal Budibase data source if you think that would help.

**Reply by antm** [reply]
- I'm using and external datasource (Supabase db).
- Using cloud version of Budibase, just updated to the latest version

- Less than works as expected. 
- Greater than works as expected. 
- _**Equal to** returns no matches when expected._

Here's how my filters are set:
![image](https://github.com/Budibase/budibase/assets/191669/2690354d-0742-4c00-b828-983d610ed52a)

I have a container called "Filter". So my object is called Filter.Fields..... Aside from that I think it's the same setup.

**Reply by ConorWebb96** [reply]
I'm not seeing any instance of you using equals in any of the screenshots you've sent. Does the `like` filter work? This is likely an issue with Superbase db. As i've tried this in both Postgres and MySQL and they both seem to work fine.

I hope this helps.
